daffodil-cli/DISCLAIMER

```diff
@@ -0,0 +1,7 @@
+Apache Daffodil is an effort undergoing incubation at The Apache Software
```

Comment:
The disclaimer should only have to appear once, in the source root.  If this is the root of the binary artifact that's fine, but its ideal if it can be a single entry.

Reply:
This was done because the daffodil-cli directory is a subproject, and there are some cases where SBT will prevent one from pulling in files outside of a subdirectory. This might not be one of those cases, I'll see if there is a way to only have one file.